org.midori_browser.Midori.json

```diff
@@ -0,0 +1,50 @@
+{
+  "app-id": "org.midori_browser.Midori",
+  "branch": "stable",
+  "runtime": "org.gnome.Platform",
+  "runtime-version" : "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "midori",
+  "finish-args" : [
+    "--device=dri",
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-download:rw",
+    "--talk-name=org.freedesktop.Notifications",
+    "--system-talk-name=org.freedesktop.GeoClue2",
+    "--talk-name=org.freedesktop.secrets"
+  ],
+  "rename-icon": "midori",
+  "rename-appdata-file": "midori.appdata.xml",
```

Comment:
Have you spoke with upstream about following the naming recommendations of the specs?

Reply:
Upstream is reading the comments now 😛️

I guess we could change that. It seems a bit strange to do with distro packages, but there's no real reason against renaming this file.

Reply:
Although renaming the icon would probably break themes in the short term

Reply:
https://github.com/midori-browser/core/pull/343

Reply:
does the icon need to follow such name convention of org.midori_browser.Midori.svg? I usually don't see icons named this way

Reply:
Yes icons must also.